pkg/standalone-utils/contracts/PoolSwapFeeHelper.sol

```diff
@@ -0,0 +1,106 @@
+                                    Manage Pools
+    ***************************************************************************/
+
+    /// @inheritdoc IPoolSwapFeeHelper
+    function addPools(address[] calldata newPools) external authenticate {
+        uint256 length = newPools.length;
+        IVault vault = getVault();
+
+        for (uint256 i = 0; i < length; i++) {
+            address pool = newPools[i];
+
+            // Ensure the address is a valid pool.
+            if (vault.isPoolRegistered(pool) == false) {
+                revert IVaultErrors.PoolNotRegistered(pool);
+            }
+
+            // Pools cannot have a swap fee manager.
+            if (vault.getPoolRoleAccounts(pool).swapFeeManager != address(0)) {
+                revert PoolHasSwapManager(pool);
+            }
+
+            if (_pools.add(pool) == false) {
+                revert PoolAlreadyInSwapFeeSet(pool);
+            }
+
+            emit PoolAddedToSwapFeeSet(pool);
+        }
+    }
+
+    /// @inheritdoc IPoolSwapFeeHelper
+    function removePools(address[] memory pools) public authenticate {
+        uint256 length = pools.length;
+        for (uint256 i = 0; i < length; i++) {
+            address pool = pools[i];
+            _ensureKnownPool(pool);
+            _pools.remove(pool);
+
+            emit PoolRemovedFromSwapFeeSet(pool);
+        }
+    }
+
+    /// @inheritdoc IPoolSwapFeeHelper
+    function setStaticSwapFeePercentage(address pool, uint256 swapFeePercentage) public authenticate {
+        _ensureKnownPool(pool);
+
+        getVault().setStaticSwapFeePercentage(pool, swapFeePercentage);
+    }
+
+    /***************************************************************************
+                                    Getters                                
+    ***************************************************************************/
+
+    /// @inheritdoc IPoolSwapFeeHelper
+    function getPoolCount() external view returns (uint256) {
+        return _pools.length();
+    }
+
+    /// @inheritdoc IPoolSwapFeeHelper
+    function hasPool(address pool) external view returns (bool) {
+        return _pools.contains(pool);
+    }
+
+    /// @inheritdoc IPoolSwapFeeHelper
+    function getPools(uint256 from, uint256 to) public view returns (address[] memory pools) {
+        uint256 poolLength = _pools.length();
+        if (from > to || to > poolLength || from >= poolLength) {
+            revert IndexOutOfBounds();
+        }
+
+        pools = new address[](to - from);
+        for (uint256 i = from; i < to; i++) {
+            pools[i - from] = _pools.at(i);
+        }
+    }
+
+    /***************************************************************************
+                                Internal functions                                
+    ***************************************************************************/
+
+    function _ensureKnownPool(address pool) internal view {
```

Comment:
how about `ensureAddedPool` / `ensurePoolAdded`? Would be consistent with `addPools`.